vpd-manager/src/parser.cpp

```diff
@@ -56,111 +58,144 @@ types::VPDMapVariant Parser::parse()
 
 int Parser::updateVpdKeyword(const types::WriteVpdParams& i_paramsToWriteData)
 {
-    // Update keyword's value on hardware
     int l_bytesUpdatedOnHardware = -1;
     try
     {
-        std::shared_ptr<ParserInterface> l_vpdParserInstance =
-            getVpdParserInstance();
-        l_bytesUpdatedOnHardware =
-            l_vpdParserInstance->writeKeywordOnHardware(i_paramsToWriteData);
-    }
-    catch (const std::exception& l_exception)
-    {
-        logging::logMessage(
-            "Error while updating keyword's value on hardware path " +
-            m_vpdFilePath + ", error: " + std::string(l_exception.what()));
-        // TODO : Log PEL
-        return -1;
-    }
+        // Update keyword's value on hardware
+        try
+        {
+            std::shared_ptr<ParserInterface> l_vpdParserInstance =
+                getVpdParserInstance();
+            l_bytesUpdatedOnHardware =
```

Comment:
not using constants SUCCESS/FAILURE to initialise this variable.

Reply:
Done